src/hotspot/os_cpu/linux_aarch64/orderAccess_linux_aarch64.hpp

```diff
@@ -30,6 +30,8 @@
 
 #include "runtime/vm_version.hpp"
 
+#define inlasm_isb()    asm volatile("isb" : : : "memory")
+
 // Implementation of class OrderAccess.
```

Comment:
This #define of inlasm_isb() looks wrong. Surely it should be in the body of OrderAccess::cross_modify_fence_impl() given that it's not used anywhere else. All the extra indirection does is confuse the reader.

Reply:
Agreed. It was designed to fit with my patch which did the same for the dmb's - but I've closed that patch. Will fix.

Reply:
Updated